<launch>
	<node pkg="dji_sdk" type="dji_sdk_node" name="dji_sdk" output="screen">
		<!-- node parameters -->
		<!-- For Drone Version enter M100 or A3_31 or A3_32 depending on your Flight Controller/Aircraft -->
                <param name="drone_version" type="string" value="M100"/>
		<param name="serial_name" type="string" value="/dev/ttyUSB0"/>
		<param name="baud_rate" type="int" value="921600"/>
		<param name="app_id" type="int" value="1040678"/>
		<param name="app_version" type="int" value="1"/>
		<param name="app_bundle_id" type="string" value="Welcome to use dji-sdk"/>
		<param name="enc_key" type="string" value="a41335b074411eb3408029cd0955324df2d3b84e5c0d80ed8344ae5c34e05cb2"/>
		<param name="groundstation_enable" type="int" value="1"/>
	</node>
</launch>
